packages/cable/package.py

```diff
@@ -32,7 +32,11 @@ def url_for_version(self, version):
 
     variant(
         "library",
-        default=False,
+        default="none",
+        values=(
+            "none",
+            "ESM1.6"
```

Comment:
I would prefer the value "access-esm1.6" to be used, to match the other packages, unless there is a good reason to use "ESM1.6".